cmd/crud/handlers/user.go

```diff
@@ -28,13 +29,14 @@ func check(err error) error {
 
 // User represents the User API method handler set.
 type User struct {
+	Log      *log.Logger
```

Comment:
This field is also unused, same as on `type Health`.

Reply:
Brilliant